File: Daily Challenges/2021/May/Day 3/Running-Sum-Of-1D-Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
    var s = [];
    s.push(nums[0])
    for(var i = 1; i<nums.length; i++){
        s.push(s[s.length-1]+nums[i]);
    }
    return s;
};

File: Daily Challenges/2021/May/Day 3/Running-Sum-Of-1D-Array.cs
public class Solution {
    public int[] RunningSum(int[] nums) {
        int[] s = new int[nums.Length];
        s[0] = nums[0];
        for(int i = 1; i<nums.Length; i++){
            s[i] = s[i-1]+nums[i];
        }
        return s;
    }
}
